srcalc/src/main/java/gov/va/med/srcalc/util/VistaFieldTranslation.java

```diff
@@ -0,0 +1,19 @@
+package gov.va.med.srcalc.util;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class is devoted to translating any information that comes from Vista
+ * to element values present on the web pages (values that come from the srcalc database).
+ */
+public class VistaFieldTranslation {
```

Comment:
I think putting these constants in `VistaPatientDao` would be better encapsulation. Want to discuss?